website/README.md

```diff
@@ -1,13 +1,15 @@
 # Focalboard website
 
-This folder is used to manage the website at [focalboard.com](https://www.focalboard.com/). It was built using [Hugo](https://gohugo.io/).
+This folder is used to manage the website at [focalboard.com](https://www.focalboard.com/). It was built using [Hugo](https://gohugo.io/). The website 
```

Comment:
Sentence fragment dangling at the end there.

Reply:
Thank you for picking this up!